src/build_backend.py

```diff
@@ -86,7 +86,7 @@ def write(filename: str, data: AnyStr) -> None:
 
         def beipack_self(main: str, args: str = '') -> bytes:
             from cockpit._vendor.bei import beipack
-            contents = {name: wheel.read(name) for name in wheel.namelist()}
+            contents = {name: wheel.read(name) for name in sorted(wheel.namelist())}
```

Comment:
I think this is the wrong spot.  I imagine the non-deterministic ordering comes from the `os.walk()` call in `find_sources()`.  Then the files get added to the wheel in a non-deterministic order, and we iterate the wheel (getting the same order, I'd assume).

Two possible ideas for a fix:

 - change `find_sources()` to no longer be a generator but to build a list internally, calling `.sort()` on it before returning it; or
 - (and this is the better one, I think): add `sorted()` around the two invocation sites of `find_sources()` where it matters: in `build_wheel()` and `build_sdist()`.  For `find_copy()` it's irrelevant.